ipn/ipnlocal/c2n.go

```diff
@@ -204,9 +203,8 @@ func (b *LocalBackend) handleC2NUpdatePost(w http.ResponseWriter, r *http.Reques
 	}
 
 	cmd := exec.Command(cmdTS, "update", "--yes")
-	buf := new(bytes.Buffer)
-	cmd.Stdout = buf
-	cmd.Stderr = buf
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
```

Comment:
without any sort of prefix on these, won't this be a little mysterious mixed in with the other stdout/stderr?

can you put this behavior behind a default false envknob maybe?

Reply:
It's not really needed anymore after I started writing output to a file, reverted.